src/utils_general/statistics.py

```diff
@@ -5,9 +5,20 @@
 from scipy.stats import genextreme as gev
 import matplotlib.pyplot as plt
 
+
 def get_return_period_function_analytical(
-    df_rp: pd.DataFrame, rp_var: str, show_plots: bool, plot_title = "",
+    df_rp: pd.DataFrame,
+    rp_var: str,
+    show_plots: bool = False,
+    plot_title: str = "",
 ):
+    """
+    :param df_rp: DataFrame where the index is the year, and the rp_var column contains the maximum value per year
+    :param rp_var: The column with the quantity to be evaluated
+    :param show_plots: Show the histogram with GEV distribution overlaid
+    :param plot_title: The title of the plot
+    :return: Interpolated function that gives the quantity for a give return period
```

Comment:
So great to have these docstrings :)

Reply:
I think we can make them a requirement with some extensions to flake-8... a future issue :)